.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+Use this template to give a detailed message describing the change you want to make to the code.
+If you are unclear on what should be written here, see  https://github.com/wrf-model/WRF/wiki/Making-a-good-pull-request-message for some general guidance
```

Comment:
I wonder if we should copy this over as well and modify.  Probably not a bad idea to build out our wiki for some dev docs anyway 🤷‍♀ .

Reply:
I agree that we should have our own wiki. If we don't want to delay this PR, though, we could just remove this line for now.

Reply:
Sounds good to me.